ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/TribuoUtil.java

```diff
@@ -95,6 +134,9 @@ public static <T extends Output<T>> MutableDataset<T> generateDataset(DataFrame
                     // TODO: support anomaly labels to evaluate prediction result
                     example = new ArrayExample<>((T) new Event(defaultEventType), featureNamesValues.v1(), featureNamesValues.v2()[i]);
                     break;
+                case LABEL:
+                    example = new ArrayExample<>((T) new Label(""), featureNamesValues.v1(), featureNamesValues.v2()[i]);
```

Comment:
You can use [`UNKNOWN_LABEL`](https://tribuo.org/learn/4.2/javadoc/org/tribuo/classification/LabelFactory.html#UNKNOWN_LABEL) for this if you want. It reduces the amount of allocation, and is fine to be shared among examples at prediction time. Or `outputFactory.getUnknownOutput()` as you already have an output factory reference.

Reply:
Replaced with outputFactory.getUnknownOutput().